spec/models/micropost_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Micropost, type: :model do
+  
+
+  before do
+    user = User.create(
+      name: "yamada",
+      email: "hoge@google.com",
+      profile: "I work for my life.",
+      url: "https://github.com/hogehoge",
+      password: "foobarfoobar"
+    )
+    @micropost = user.microposts.build(
+      content: "This is a sample micropost."
+    )
+  end
+
+  it "is valid with a content, and  user id" do
+    expect(@micropost).to be_valid
+  end
+
+  it "is invalid without a user id" do
+    @micropost.user_id = nil
+    @micropost.valid?
```

Comment:
```suggestion
    expect(@micropost).to be_invalid
```

または `expect(@micropost).not_to be_valid` でもOK